cli/Cargo.toml

```diff
@@ -49,6 +49,17 @@ whoami = "0.7.0"
 path = "../libsplinter"
 features = ["admin-service", "registry"]
 
+[dependencies.scabbard]
```

Comment:
typo in commit message: require -> required

Reply:
Fixed